### PR TITLE
Requirements: set min require CMake to 3.16

### PR DESCRIPTION
**Pull request by Winfidonarleyan**
## Related Issue 
https://github.com/azerothcore/azerothcore-wotlk/pull/5082

### CODE REVIEW

File: docs/Requirements.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ Than **reboot** to sync the bash and use cmake from the terminal, than type the
 cmake --version
 ```
 
-You should have a version **>= 3.8**.
+You should have a version **>= 3.16**.
```
Comment (FrancescoBorzi) [inline docs/Requirements.md:58]: good, check line `127` as well

Comment (Winfidonarleyan) [reply docs/Requirements.md:58]: I did this pr from my smartphone :D

Comment (Winfidonarleyan) [reply docs/Requirements.md:58]: And thanks for catch

**Comment by Winfidonarleyan** [comment]
Need merge this

**Review by Helias — APPROVED** [review]

**Pull request merged**